pom.xml

```diff
@@ -20,7 +20,7 @@
 
   <groupId>io.cdap.plugin</groupId>
   <artifactId>hydrator-plugins</artifactId>
-  <version>2.3.3</version>
+  <version>2.3.4</version>
```

Comment:
Missing the `cdap.version` bump in this file

Reply:
updated